runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/state/WindmillMap.java

```diff
@@ -142,6 +142,7 @@ protected Windmill.WorkItemCommitRequest persistDirectly(WindmillStateCache.ForK
           .setTag(keyBytes)
           .setStateFamily(stateFamily)
           .getValueBuilder()
```

Comment:
nit: I think you can fix instead by removing the valuebuilder and settimestamp and it's simpler code.

Reply:
Thanks. I fixed it to remove the code from `getValueBuilder()`.